aws/resource_aws_default_vpc_test.go

```diff
@@ -26,13 +26,50 @@ func TestAccAWSDefaultVpc_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"aws_default_vpc.foo", "tags.%", "1"),
 					resource.TestCheckResourceAttr(
-						"aws_default_vpc.foo", "tags.Name", "Default VPC"),
-					resource.TestCheckNoResourceAttr(
-						"aws_default_vpc.foo", "assign_generated_ipv6_cidr_block"),
-					resource.TestCheckNoResourceAttr(
-						"aws_default_vpc.foo", "ipv6_association_id"),
-					resource.TestCheckNoResourceAttr(
-						"aws_default_vpc.foo", "ipv6_cidr_block"),
+						"aws_default_vpc.foo", "tags.Name", "terraform-testacc-default-vpc"),
+					// For backwards compatability the assign_generated_ipv6_cidr_block attribute has no default value
+					// for a region's default VPC. The assumption here is that IPv6 is not enabled.
+					testAccCheckVpcIpv6(&vpc, false),
```

Comment:
This fails or can be flakey (at least in our parallelized acceptance testing environment):

```
=== RUN   TestAccAWSDefaultVpc_basic
--- FAIL: TestAccAWSDefaultVpc_basic (80.38s)
    testing.go:518: Step 0 error: Check failed: Check 6/6 error: Unexpected IPv6 CIDR block: 2600:1f14:c5e:af00::/56
FAIL
```

I think it should probably be removed from the acceptance tests except where explicitly enabled/disabled.

Reply:
Agreed. Will fix.